whois-api/src/main/java/net/ripe/db/whois/api/httpserver/RewriteEngine.java

```diff
@@ -87,19 +86,11 @@ public RewriteHandler getRewriteHandler() {
         final VirtualHostRuleContainer syncupdatesVirtualHostRule = new VirtualHostRuleContainer();
         syncupdatesVirtualHostRule.addVirtualHost(syncupdatesVirtualHost);
         rewriteHandler.addRule(syncupdatesVirtualHostRule);
-
-        final RewriteRegexRule syncupdatesEmptyQueryStringRule = new RewriteRegexRule(
```

Comment:
I am checking the history of GIT and this has been there from 05/2022
Do you know why it started breaking now?

Reply:
ok, it is in the description